include/onnxruntime/core/graph/graph.h

```diff
@@ -578,6 +579,15 @@ class Graph {
   /** Gets the path of the owning model, if any. */
   const Path& ModelPath() const;
 
+  /** Returns true if this is a subgraph or false if it is a high-level graph. */
+  bool IsSubgraph() const { return parent_graph_ != nullptr; }
```

Comment:
Many functions have been re-ordered in this file. Haven't looked very closely. I suspect it's to support the new ifdef?

Reply:
Re-ordering was to group things that fall under the same #ifdef's without moving them too far. Just trying to reduce the number of different ifdef blocks.